docs/content/contentlist.md

```diff
@@ -0,0 +1,444 @@
+# All Things Alephium List
```

Comment:
Every page should start with:
1. Frontmatter
2. Snippet for untranslated text

Check out how another page is done, for example this one: https://github.com/alephium/docs/blob/master/docs/glossary.md?plain=1#L1-L10

Reply:
Fixed